File: src/main/java/no/nav/domain/pensjon/regler/vedtak/InntektEtterUforhet.java
package no.nav.domain.pensjon.regler.vedtak;

import java.util.Date;

/**
 * Angir inntekt etter uførhet (IEU).
 *
 * @author Magnus Bakken (Accenture), PK-9491, PK-10597 PKPYTON-1063
 * @author Steinar Hjellvik (Decisive), PK-6172
 */
public class InntektEtterUforhet extends AbstraktBeregningsvilkar {

    /**
     * Inntekten.
     */
    private int inntekt;

    /**
     * Virkningstidspunktet for inntekt etter uførhet.
     */
    private Date ieuDato;

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/vedtak/InntektEtterUforhet.kt
package no.nav.pensjon.regler.domain.vedtak

import java.util.*
import java.time.LocalDate

/**
 * Angir inntekt etter uførhet (IEU).
 */
class InntektEtterUforhet : AbstraktBeregningsvilkar() {
    /**
     * Inntekten.
     */
    var inntekt = 0

    /**
     * Virkningstidspunktet for inntekt etter uførhet.
     */
    @Deprecated("Use ieuDatoLd instead")
    var ieuDato: Date? = null
    var ieuDatoLd: LocalDate? = null

}